CHANGELOG.md

```diff
@@ -5,17 +5,26 @@ Notable changes to this project are documented in this file. The format is based
 ## [Unreleased]
 
 Breaking changes:
+- Added support for PureScript 0.14 and dropped support for all previous versions (#133)
+- Replaced `ComonadEnv` constraint with `ComonadAsk` in the `asks` function (#131)
```

Comment:
This is _barely_ a breaking change, but as discussed in #131 it can be breaking.